notebook/tests/selenium/utils.py

```diff
@@ -100,6 +100,20 @@ def focus_cell(self, index=0):
         self.to_command_mode()
         self.current_cell = cell
 
+    def find_and_replace(self, index=0, find_txt='', replace_txt='', replace_all=False):
+        self.focus_cell(index)
+        esc(self.browser, 'F')
+        time.sleep(1)    # TODO: find better way to fill the find and replace form
```

Comment:
Are there any specific elements on the form? We can wait for an element to appear.

Reply:
Hmm, I don't want to comment now before trying again (I rem trying the wait logic while working on this, but couldn't get the wait working because I couldn't find any specific element, but let me retry)